client/src/lib.rs

```diff
@@ -928,6 +942,44 @@ impl WorkflowClientTrait for Client {
             .into_inner())
     }
 
+    async fn signal_with_start_workflow_execution(
+        &self,
+        input: Option<Payloads>,
+        task_queue: String,
+        workflow_id: String,
+        workflow_type: String,
+        options: WorkflowOptions,
+        signal_name: String,
+        signal_input: Option<Payloads>,
+    ) -> Result<SignalWithStartWorkflowExecutionResponse> {
+        let request_id = Uuid::new_v4().to_string();
+        Ok(self
+            .wf_svc()
+            .signal_with_start_workflow_execution(SignalWithStartWorkflowExecutionRequest {
+                namespace: self.namespace.clone(),
+                workflow_id,
+                workflow_type: Some(WorkflowType {
+                    name: workflow_type,
+                }),
+                task_queue: Some(TaskQueue {
+                    name: task_queue,
+                    // #FIXME the go sdk gets this from a proto enum.  Not sure where
+                    // this is available in the rust prost generated stuff is at the moment.
```

Comment:
Great.  Found it.  I was looking at the ```start_workflow``` function as a reference and it was using 0 instead of the enum.  Didn't notice the other functions were though.  Fixed both.